src/elements/content-sharing/ContentSharing.js

```diff
@@ -30,12 +29,17 @@ import type {
     SharedLinkUpdateSettingsFnType,
 } from './types';
 
+import '../common/base.scss';
+import '../common/fonts.scss';
+import '../common/modal.scss';
+
 type ContentSharingProps = {
     apiHost: string,
     displayInModal?: boolean,
     itemID: string,
     itemType: ItemType,
     language: string,
+    messages?: StringMap,
```

Comment:
Will these always be `FormattedMessages`? If so, can we use `MessageDescriptor` from `react-intl` here?

Reply:
The messages get passed in as a map of id => text, so I don't think MessageDescriptor will work